### ISSUE TITLE
Determine NitroAdjudicator address based on asset holder

### ISSUE THREAD

**Issue by snario** [statechannels/statechannels#3024]
We actually don't need to provide the Nitro Adjudicator address at all to the wallet, I don't think. We can just query it when we need it by calling `AdjudicatorAddress()` on whatever asset is being used.

**Comment by lalexgap** [comment]
I don't think we want to do this now but it might be worth doing later if we're in the area.
